### ISSUE TITLE
translation - packing list

### ISSUE THREAD

**Issue by mr-jingles** [firegento/firegento-pdf#260]
Currently the packing list is translated as "Shipment". I am referring to  line 483 in Abstract.php

} elseif ($mode == 'shipment') {
      $title = 'Shipment';
} else {

**Comment by Schrank** [comment]
Correct, what is the issue?

**Comment by sprankhub** [comment]
I think the issue is that most shops use this PDF as a packing slip. "Shipment" is a weird title. In any case, I think we should stick to Magento's default title. The default title is indeed "Packzettel" in German (I did not check in English, but it is most likely packing slip).

**Comment by mr-jingles** [comment]
@sprankhub - that is what I meant. I think the label should be packing slip and not shipment.

**Comment by sprankhub** [comment]
Alright, this is implemented now. Thanks for bringing that up @mr-jingles!

@Schrank should we also change the default filename of the downloaded PDFs from shipment* to packingslip*? I think this would be consequent.

**Comment by pquerner** [comment]
Our shop owner said "Packzettel" is not what it should be (not saying "illegal" as I am no lawyer and didnt have anything in writing to back me up). He said it should be "Lieferschein". Should we fix this by overwriting this in a CSV or fix it again in PHP ? @sprankhub

**Comment by Schrank** [comment]
I have no clue what is right here. I think just putting it into translate.csv fixes the problem for you. And about the question of sprankhub: Ich would change the default.

**Comment by sprankhub** [comment]
I am not sure if there are legal requirements for packingslips. I decided to use "Packzettel", because this translation is also used in the DE community translation: https://github.com/riconeitzel/German_LocalePack_de_DE/blob/8249c5856c001ac9a9cf1b35a6764d41c0450c2e/app/locale/de_DE/Mage_Sales.csv#L488

I think IF this is changed, it should be changed in both projects...

**Comment by pquerner** [comment]
> Einen Packzettel / manchmal auch Pickliste genannt ist das was die Logistiker / Bearbeiter zur Zusammenstellung der Ware nutzen bevor diese an den Kunden geht, der Kunde wiederum erhält bei Lieferung einen Lieferschein beigefügt.
> 
> Für mich besteht hier schon ein Unterschied, da der Lieferschein die tatsächlich gelieferte Menge aufführt, ggf. auch Vermerke wenn ein Artikel in der bestellten Menge nachgeliefert wird oder so. Es ist ja möglich das während der Kommission festgestellt wird das etwas so wie auf dem Packzettel aufgeführt nicht geliefert werden kann.

http://forum.azmagento.com/how-to/lieferschein-vs-packzettel-63-nbsp--39851.html#forumcm

> Pickzettel (auch Pickliste oder Packzettel, in der Schweiz Rüstliste; von englisch pick für auswählen) werden in Unternehmen mit großen und umfangreichen Lagerbeständen dazu verwendet, um die Paketinhalte für den Versand zusammenzustellen. Das Packen geschieht in diesem Fall nach der Art eines typischen Laufzettels.

> Dazu wird neben einem Lieferschein – dieser gibt an, welche Waren geliefert werden – und der Rechnung – gibt an, welche Waren bezahlt werden müssen – ein „Pickzettel“ erstellt, auf dem für den Lagerarbeiter aufgelistet ist, welche Waren in welchem Lager an welchem Ort (Regal, Stellage usw.) vorrätig sind und von dort in das Paket mit übernommen werden müssen.

https://de.wikipedia.org/wiki/Pickzettel

Pickzettel/Packzettel verstehe ich in dem Zusammenhang also anders, als einen Lieferschein, den Magento dir ja ausdruckt.

(Sorry for german, but I think this just concerns german speaking users)

**Comment by sprankhub** [comment]
I also think that "Lieferschein" is better, but I would like to stick to the community translation. @riconeitzel what do you think - can we change the wording in both projects? That would be the best solution imho.

**Comment by riconeitzel** [comment]
Yes, I once had "lieferschein" but I don't remember why I changed it again.

**Comment by sprankhub** [comment]
@riconeitzel would you agree that "Lieferschein" is more reasonable than "Packzettel"? Can we change it in both projects?

**Comment by sprankhub** [comment]
@riconeitzel bump ;-)

**Comment by Schrank** [comment]
@riconeitzel don't overlook this.
